### PR TITLE
MIJN-999 Disabled responsive css for IE11

### PR DESCRIPTION
**Pull request by BennievanderWel**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "axios": "^0.19.0",
     "axios-mock-adapter": "^1.16.0",
     "classnames": "^2.2.6",
+    "detect-browser": "^4.6.0",
```
Comment (timvanoostrom) [inline package.json]: Maybe use something more lightweight? like:

```
https://stackoverflow.com/questions/19999388/check-if-user-is-using-ie
```

File: src/hooks/media.hook.ts [hunk 2]
```diff
@@ -1,15 +1,22 @@
 import { Breakpoints } from './../App.constants';
 // Helper functions to determine screen size in JS
 import { useMedia } from 'use-media';
+import { detect } from 'detect-browser';
+
+const isIE11 = detect()!.name === 'ie';
+
+export function useIE11() {
```
Comment (timvanoostrom) [inline src/hooks/media.hook.ts]: There seems to be no need to put the check into a function

File: src/styles/mixins/mixins.scss [hunk 3]
```diff
@@ -64,3 +66,11 @@
     @content;
   }
 }
+
+@mixin media-query($sizes...) {
+  @include media($sizes...) {
+    @supports (display: grid) {
```
Comment (timvanoostrom) [inline src/styles/mixins/mixins.scss]: Are you sure all of our other browsers support grid and the @supports tag?

File: src/hooks/media.hook.ts (hunk 2 above)
Comment (timvanoostrom) [reply src/hooks/media.hook.ts]:
```
var ua = window.navigator.userAgent;
var isIE = /MSIE|Trident/.test(ua);

if ( isIE ) {
  //IE specific code goes here
}
```

File: src/styles/mixins/mixins.scss (hunk 3 above)
Comment (BennievanderWel) [reply src/styles/mixins/mixins.scss]: Yes

File: src/hooks/media.hook.ts (hunk 2 above)
Comment (BennievanderWel) [reply src/hooks/media.hook.ts]: True

File: package.json (hunk 1 above)
Comment (BennievanderWel) [reply package.json]: Done

**Pull request merged**